components/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java

```diff
@@ -562,12 +695,20 @@ private void setWReplyUrl(PassiveSTSHttpServletRequestWrapper request) {
                 if (PASSIVE_STS_CLIENT_TYPE.equals(inboundAuthenticationConfig.getInboundAuthType())) {
                     Property[] properties = inboundAuthenticationConfig.getProperties();
                     if (ArrayUtils.isNotEmpty(properties)) {
+                        String wreplyUrl = null;
```

Comment:
why do we need to set null?

Reply:
The variable needs to be initialized with null since we are using StringUtils.isNotBlank(wreplyUrl)[1] at a later step.

[1] https://github.com/wso2-support/identity-inbound-auth-sts/blob/support-5.5.1.x-full/components/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java#L559